tests/test___main__.py

```diff
@@ -23,7 +23,17 @@
                "x-github-delivery": "1234"}
     # Sending a payload that shouldn't trigger any networking, but no errors
     # either.
-    data = {"action": "closed"}
+    data = {
```

Comment:
This test is now failing, probably because closing a PR now triggers networking calls (see the comment in line 24). It probably needs to perform an action that isn't registered with the router to perform the same intent in testing.